gen/calc.py

```diff
@@ -70,6 +81,14 @@ def calculate_ip_detect_public_contents(ip_detect_contents):
     return ip_detect_contents
 
 
+def calculate_rexray_config_contents(rexray_config_filename):
+    try:
+        with open(rexray_config_filename, encoding='utf-8') as f:
+            return yaml.dump(f.read())
+    except IOError as err:
+        raise Exception('REX-Ray config file {}: {}'.format(rexray_config_filename, err))
```

Comment:
Adding in context of what the filename is, since this will show up to end users as a warning / error message.

We should actually probably be catching / re-wrapping exceptions, and do a raise from...

@branden Could you make a follow up PR to fix (I want to land this train, don't think this is blocking)